analysers/analyser_osmosis_powerline.py

```diff
@@ -421,6 +532,17 @@ def analyser_osmosis_diff(self):
         self.run(sql51)
         self.run(sql52, self.callback50)
 
+    def __callback80_fix(self, res):
```

Comment:
My fix strategy is probably not good, as it produces results like:
```xml
<error class="8">
<node id="993023726" lat="50.5270043" lon="3.0664694000000003" version="2" user="InfosReseaux">
<tag k="power" v="tower" />
<tag k="operator" v="RTE" />
<tag k="design:ref" v="L1" />
<tag k="line_attachment" v="anchor" />
<tag k="operator:wikidata" v="Q2178795" />
</node>
<location lat="50.5270043" lon="3.0664694" />
<fixes>
<fix>
<node id="993023726">
<tag action="create" k="line_management" v="branch" />
</node>
</fix>
<fix>
<node id="993023726">
<tag action="modify" k="line_management" v="branch" />
</node>
</fix>
</fixes>
</error>
```

How can I prevent `modify` item when tag is created in the same warning?

Reply:
Found out, now fixed